Source/NSOperation.m

```diff
@@ -533,6 +533,56 @@ - (void) _finish
 
 @end
 
+	  
+@implementation NSBlockOperation
+
+// Initialize
+- (id) init
+{
+  self = [super init];
+  if(self != nil)
+    {
+      _executionBlocks = [[NSMutableArray alloc] initWithCapacity: 10];
```

Comment:
I think 1 would be a more likely capacity, as in most cases we will have only one block.